.pylintrc

```diff
@@ -11,4 +11,7 @@ format=colorized
 
 [basic]
 const-rgx=[a-z_]*|[A-Z_]*
-docstring-min-length=3
\ No newline at end of file
+docstring-min-length=3
+
+[design]
+max-attributes = 10
```

Comment:
This was needed because the `_ConnectionInfo` class has more than 7 attributes. Seems fine to bump it up a bit, IMHO.